test/e2e/dedicated_admin_test.go

```diff
@@ -0,0 +1,74 @@
+//+build e2e
+
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	_ "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var _ = Describe("Openshift on Azure dedicated-cluster-admin e2e tests [DedicatedAdmin]", func() {
+	defer GinkgoRecover()
+
+	BeforeEach(func() {
+		namespace := nameGen.generate("e2e-test-")
```

Comment:
None of the tests seem to need their own namespace so both BeforeEach and AfterEach here can be dropped at least for now.

Reply:
ah you seem to be using it to check for non-infra ns access. Probably fine for now, eventually yes we want to get that ns created by a different kubeconfig.

Reply:
actually since it's a single test where you need to create a namespace I would prefer you do the project request inside that test and remove these so we will not incur extra time from project creation and teardown on other tests.

Reply:
Done.